Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -159,8 +172,8 @@ class BackForwardListViewController: UIViewController,
 
     override func willTransition(to newCollection: UITraitCollection, with coordinator: UIViewControllerTransitionCoordinator) {
         super.willTransition(to: newCollection, with: coordinator)
-        guard let bvc = self.bvc else { return }
-        if bvc.shouldShowToolbarForTraitCollection(newCollection) != snappedToBottom, !bvc.isBottomSearchBar {
+        guard let _ = browserViewControllerFrameInfoProvider else { return }
```

Comment:
We need `guard let _ = browserViewControllerFrameInfoProvider else { return }` still? Also, if we do I think we should have `guard browserViewControllerFrameInfoProvider != nil else { return }` instead otherwise this will generate a new Swiftlint warning I think

Reply:
Ah ok yes you're right it can be removed